lessons/lesson26/lecture.md

```diff
@@ -21,32 +21,72 @@ description: «Разбор базовых приемов разработки 
 
 <!-- v -->
 
-Примитивные типы:
+Примитивные, простые, базовые типы:
 
 - string
 - number
 - boolean
-- null
-- undefined
+- void
+- unknown
+- symbol
+- any
+- null / undefined
+- never
+
+<!-- v -->
+
+Примитивные, простые, базовые типы:
+
+- array
+- tuple (кортеж)
+- enum
+- object
+- function
+
+<!-- v -->
+
+Литеральные типы
+
+```ts [1-30]
+type Yes = "yes";
+type Zero = 0;
+```
 
 <!-- v -->
 
 Литеральные типы
 
 ```ts [1-30]
 type Yes = "yes";
+let answer: Yes = "no"; // ERROR Type '"no"' is not assignable to type '"yes"'
+
 type Zero = 0;
+let result: Zero = 1; // ERROR Type '1' is not assignable to type '0'
 ```
 
 <!-- v -->
 
-Массивы
+Массивы  
+_между этими вариантами нет разницы_
```

Comment:
технически разницы нет, но есть разница в стайлгайдах 

например https://google.github.io/styleguide/tsguide.html#arrayt-type (именно об этом был комментарий)

Reply:
"легче читать" - это очень субъективная категория, стайлгайды об этом не говорят, там стоит разделение по типам элементов внутри массива (For simple types (containing just alphanumeric characters and dot), use the syntax sugar for arrays, T[], rather than the longer form Array<T>)

Придерживаться этого правила или нет - решает команда в начале проекта, практической разницы между ними нет.

Reply:
про практическую разницу я не спорю, но есть распространенные стайлгайды (опять же https://typescript-eslint.io/rules/array-type/ с его дефолтным поведением)